build.gradle

```diff
@@ -2,29 +2,29 @@
 
 buildscript {
     ext {
-        kotlin_version = '1.5.20'
-        gradle_version = '4.2.2'
+        kotlin_version = '1.5.31'
+        gradle_version = '7.0.2'
         google_services_version = '4.3.8'
         firebase_crashlytics_version = '2.7.1'
         firebase_bom_version = '28.0.1'
 
-        voximplant_sdk_version = '2.25.1'
+        voximplant_sdk_version = '2.27.0'
 
         material_version = '1.4.0'
-        appcompat_version = '1.3.0'
+        appcompat_version = '1.3.1'
         core_ctx_version = '1.6.0'
-        constraintLayout_version = '2.0.4'
+        constraintLayout_version = '2.1.1'
         vectordrawable_version = '1.1.0'
         preferences_ktx_version = '1.1.1'
         lifecycle_ktx_version = '2.3.1'
-        activity_ktx_version = '1.2.3'
-        cameraX_version = '1.1.0-alpha05'
+        activity_ktx_version = '1.3.1'
+        cameraX_version = '1.1.0-alpha08'
```

Comment:
1.1.0-alpha09 version is available, please check if we can update to it

Reply:
Upgrading to 1.1.0-alpha09 requires some additional code changes. The version will be updated later.